mlrun/k8s_utils.py

```diff
@@ -269,6 +271,49 @@ def get_logger_pods(self, uid, namespace=''):
 
         return results
 
+    def get_shell_pod_name(self, pod_name='shell'):
+        shell_pod = self.v1api.list_namespaced_pod(namespace=self.namespace)
+        for i in shell_pod.items:
+            if pod_name in i.metadata.name:
+                logger.info("%s\t%s\t%s" % (i.status.pod_ip, i.metadata.namespace, i.metadata.name))
+                shell_name = i.metadata.name
+                break
+        return shell_name
+
+    def exec_shell_cmd(self, cmd, v3io_access_key):
```

Comment:
@aviaIguazio access key validation should not be in this class (generic k8s) , and in general this sort of a hack is not advised and we dont want to put it in mlrun base code